src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DefaultContainerFactory.java

```diff
@@ -137,7 +157,53 @@ protected Probe createProbe(Integer externalPort, String endpoint, int timeout,
 		return cmdArgs;
 	}
 
-	/**
+    /**
+     * The list represents a single command with many arguments.
+     *
+     * @param request AppDeploymentRequest - used to gather application overridden
+     * container command
+     * @return a list of strings that represents the command and any arguments for that command
+     */
+    private List<String> getContainerCommand(AppDeploymentRequest request) {
+        List<String> containerCommandList = new ArrayList<>();
+        String containerCommand = request.getDeploymentProperties()
+                .get("spring.cloud.deployer.kubernetes.containerCommand");
+        if (containerCommand != null) {
+			logger.info("Building container command from AppDeploymentRequest: "
+					+ containerCommand);
+            String[] containerCommandSplit = containerCommand.split(",");
```

Comment:
The way Kubernetes handles container commands is in  a comma delimited format, for instance:

```
apiVersion: v1
kind: Pod
metadata:
  name: hello-world
spec:  # specification of the pod's contents
  restartPolicy: Never
  containers:
  - name: hello
    image: "ubuntu:14.04"
    command: ["/bin/echo", "hello", "world"]
```

Grabbing a comma delimited value from the spring.cloud.deployer.kubernetes.containerPorts makes it easier to pass the command and argument list to the fabric8 container model, which expects the shell command as list of strings.